soccer/Processor.hpp

```diff
@@ -177,6 +177,10 @@ class Processor : public QThread {
 
     void setFieldDimensions(const Field_Dimensions& dims);
 
+    const bool isRadioOpen();
+
+    const bool isInitialized();
```

Comment:
This could use a brief comment to give some context to what is initialized and the fact that start/run is what does it.